Data_Collection_group/Data_Collection_Readme.md

```diff
@@ -0,0 +1,18 @@
+## data Collection - Readme.md

```

Comment:
Please place this in the main folder under README.md so it can be viewed on the front page.

Reply:
Hi Professor, the requested changes have been done. Please review and approve the request.